### PR TITLE
Report json seed data

### PR DESCRIPTION
**Pull request by perrymant**
Just the JSON file this time!

Still wondering if there is a mistake in the product design because I have written (lines 48-61):

Response
200: a report statement will be sent:
```json
{
  "report": [
    {
      "date": "2018-11-28T17:34:45Z",
      "amount": 1234,
      "balance": 12345678,          <<<ISSUE???
      "description": "blah"
    }
  ]
}
```
Q: should this have the balance removed?

### CODE REVIEW

**Comment by kinbiko** [comment]
Okay, just to clarify [my previous comment here](https://github.com/perrymant/moneymaker/pull/42#pullrequestreview-191946395).
When we request a report, we read the stored account file which has a list of all the transactions. This file does *not* contain the balance. Before we return this to the user, we calculate the balance for each transaction, essentially turning it into a report line. Then we return the report, which is essentially just all the report lines.

*the response of the API when querying for report* will contain the balance. It's OK that it doesn't do that right now, because we've not implemented it completely yet.

Tl;dr: The file stored != a report response.

We do have an inaccuracy in our engineering design however, where it says that the file stored should contain the balance. I believe that to be a copy+paste typo, that should be amended.

Please chat to me in Slack before continuing if there's anything else that's unclear.
Sorry for the confusion.

**Review by kinbiko — APPROVED** [review]
